Chapters/1-3-Chapter3.md

```diff
@@ -0,0 +1,259 @@
+
+"Good luck, Liffa," said Jathi, somberly.
+
+"Don't get yourself killed," said Thimlu.
+
+---
+
+"Aey? Kipnu?" I called as I entered the clinic. He chirped sleepily from a back room and I followed the call. I came to the doorway and watched him for a moment. His sleepy face always made me swoon no matter how silly it looked.
+
+"Hey," I interrupted myself, "I'm sorry to wake you but I need to talk."
+
+He shuffled around the cot, looking confused, then looked up at my shadow in the doorway.
+
+"Liffa, that you?"
+
+He pushed himself up and took a moment to wake up. Then he pulled a lightglass from his pocket and slid it onto the power rails above him. His sleepy eyes were still partially covered by his eyehaw.
+
+"You surprised me. What brings you here?"
+
+"I'm going on a trip to the capital."
+
+"What? Why?" He asked, his eyes now completely open.
+
+"It's just something between Melna and me"
+
+"You're going all the way to the capital to settle an argument? Why can't you wait until she comes back?"
+
+"It's... a long story"
+
+"You mean you don't want to tell me"
+
+I paused, dumbstruck and scrambling for words.
+
+"You're keeping a secret. Tell me what's going on-"
+
+I backed away and stammered, "I'll tell you later. Take care of Laen-Laen, he'll be coming back home."
+
+I turned to walk out the door, but he grabbed my wrist to stop me. His eyes were pleading, worried.
+
+"We swore the Oath of Trust to each other," he reminded me. "Whatever is going on between you affects me as well."
+
+"You can't be involved."
+
+"I already am."
+
+I fought off his grip, and stood just outside the door, staring at the gaudy iron floor of his office and vying with my own thoughts.
+
+_I could run now. I wanted to run. I had this all planned out. I would deliver the message, then survive Ulukoma and come back redeemed. Kipnu and Melna would understand. In the worst case, I would get a little cold in the dark. But Kipnu? I would have to fend for him too. He doesn't know how to survive on the land_
+
+He pushed again, "If you won't tell me, then I'm going with you to find out myself."
+
+"You couldn't - you have patients to see"
+
+"They can wait."
+
+"Kipnu-"
+
+"Why can't you let me in?"
+
+"Kipnu...if you go with me, it will be a long time before you come back."
+
+"What are you talking about? The trains run four times a day."
+
+"This isn't about the trains, ok? I'm going on this journey alone and it might be a while before I come back."
+
+"Stop being cryptic. What are you planning, Liffa?"
+
+His gaze cast doubt on my resolve. How easy it would have been to say yes; to have him next to me while I interrupted a room full of intellectuals about some important science I didn't understand. To know that I wasn't alone in thinking this was the right thing.
+
+No. His guilt would be my guilt. He can't be involved in this.
+
+_It's clear._
+
+"Goodbye," I said.
+
+I hopped onto Laen and didn't look back.
```

Comment:
Prediction: does the place Lif get sentenced to look nothing like they imagined?

Reply:
hehe. You know by now, but yes. I'm glad that foreshadowing is clear, but also secretive enough.